spec/menu_spec.rb

```diff
@@ -0,0 +1,11 @@
+
+require'menu'
+describe Menu do
+
+  subject(:menu) { described_class.new }
+  # let(:menu) {double :menu}
+
+    it 'displays menu list' do
+      expect(menu.show_menu).to include(:chips => 2.00)
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)